docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+  cases, you may wish to design custom child workflow activities that perform
+  operatons on the SIP before it is extracted - for example, calculating a
+  checksum of the zipped package to check for prior duplicate ingests before
+  proceeding. When this value is set to `true` Enduro will skip the extraction
+  task at the beginning of the ingest workflow, allowing you to define when and
+  how extraction occurs in the child workflow.
+* `sharedPath`: The absolute path to the directory that Enduro should use to
+  share the SIP between the primary ingest workflow and the configured
+  preprocessing child workflow. This path is required when `enabled` is set to
+  true.
+
+#### Temporal workflow configuration for preprocessing workflow
+
+These settings ensure that Enduro can connect with [Temporal] and trigger the
+configured [child workflow].
+
+**Default values**:
+
+```toml
+[preprocessing.temporal]
+namespace = "default"
+taskQueue = "preprocessing"
+workflowName = "preprocessing"
+```
+
+* `namespace`: An internal namespace label for the temporal workflows. This only
+  needs to be changed if you expect to be running multiple different workflows
+  running at once, so they don't clash.
+* `taskQueue`: In Temporal, a [Task Queue] is  a lightweight, dynamically
+  allocated queue that one or more workers can poll for tasks. Temporal supports
+  all kinds of parallelization for a distributed application architecture, and
+  and task queues can be namespaced to support such a set-up. The default value
+  of `preprocessing` here differentiates it from the general `global` task queue
+  used by Enduro, and ensures that Enduro looks for child workflow tasks in a
+  dedicated queue.
+* `workflowName`: The name of the configured [child workflow] in Temporal.
+
+### Post-storage worfklow configuration
+
+In addition to configuring a [child workflow] for custom ingest processing
+tasks, Enduro can also run an additional child workflow for [post-storage]
+processing when properly configured.
+
+Post-storage is a phase in an ingest or preservation workflow describing all the
+preservation policy-defined tasks performed on an [AIP] following
+preservation processing and AIP storage. Post-storage task examples might
+include metadata extraction and delivery to an external system (such as an
+archival management system), AIP encrpyption or replication, and more.
+
+These settings can be used to enable and configure a post-storage child workflow
+for Enduro, which will be run after AIP creation and storage following a
+successful ingest. By default at installation this section is commented out to
+render it inactive - remove the `#` hash symbol from the start of each line to
+enable this section as you configure the values.
+
+**Default values**:
+
+```toml
+# [[poststorage]]
+# namespace = "default"
+# taskQueue = "poststorage"
+# workflowName = "poststorage"
+```
+
+* `namespace`: An internal namespace label for the temporal workflows. This only
```

Comment:
Capitalization: "Temporal"

Reply:
Still "temporal"

Reply:
This one I did in fact apparently fully miss in the first review. Just searched and fixed all occurrances for the next round